### ISSUE TITLE
Have to login again on every new page using Chrome extension

### ISSUE THREAD

**Issue by dwiehoff** [hypothesis/browser-extension#484]
### Steps to reproduce
1. Highlight a portion of the text on a site
2. Navigate to a different page
3. Attempt to highlight another portion of text

### Expected behaviour
I would expect to seamlessly continue using the extension as I've authorized myself seconds ago.

### Actual behaviour
Instead, I have to authorize myself every time I navigate to a new page.

### Browser/system information
macOS 11.1 - Brave 1.18.77

### Additional details
![image](https://user-images.githubusercontent.com/20980287/104576551-a2995280-5658-11eb-9f8c-37c9ae30a098.png)

**Comment by robertknight** [comment]
> Just realized this would be better placed under https://github.com/hypothesis/browser-extension/issues; sorry about that.

No problem! - I've transferred the issue there.

**Comment by ChristinWhite** [comment]
You can indeed whitelist the extension in Brave.

In Brave open Settings / Additional settings / Privacy and security and then under Sites that can always use cookies add [*.]bjfhmglciegochdpefhhlphglcehbmek.

**Comment by esanzgar** [comment]
Good to know, thanks!

**Comment by dwiehoff** [comment]
![image](https://user-images.githubusercontent.com/20980287/106630722-616fd080-657c-11eb-8276-5481bcbb1673.png)
Does this look right to you? Doesn't seem to have any effect for me.

**Comment by ChristinWhite** [comment]
Yep, that's what I did @dwiehoff, see [this page](https://via.hypothes.is/https://web.hypothes.is/help/how-long-do-i-remain-logged-in-to-hypothesis/#annotations:_P7Gzl6REeucM9-TpP7NvQ) and [my annotation](https://hypothes.is/users/ChristinWhite?q=tag%3AHypothes.is#) for the specific directions I followed for Brave.
